controllers/admin/AdminShoppingfeedGeneralSettings.php

```diff
@@ -846,6 +846,7 @@ public function displayAjaxGetCategoryList()
             ->from('category_lang', 'cl')
             ->leftJoin('category', 'c', 'cl.id_category = c.id_category')
             ->where('cl.id_lang = ' . (int) $this->context->language->id)
+            ->where('cl.id_shop = ' . (int) $this->context->shop->id)
```

Comment:
I think a group by id_category should be better, doesn't it ?

Reply:
it was replaced with 'group by'